services/web-app/src/components/header/header.tsx

```diff
@@ -0,0 +1,17 @@
+"use client"
+import Image from "next/image";
+import { LoginButton } from '@/components/buttons/login';
+
+export const Header = (): JSX.Element => {
+  return (
+    <header className="flex flex-row navbar justify-between items-center">
+      <Image src="/icon.png" alt={"orion logo"} width={100} height={100} />
+      <h1 className="text-4xl font-mono">
+          Code Review GPT
+      </h1>
+      <LoginButton />
```

Comment:
Could you change the login button to the user's email once that user has logged in ? Right now it's really hard to see if we're logged in or not (if that's a lot of work we can create a separate ticket for that)

Reply:
I'll add this to my ticket in daily goals as it involves doing something similar with a sign out button :)